### PR TITLE
multiple bug fixes

### PR DESCRIPTION
**Pull request by gutmann**
Maximum standard deviation of 20 was only being checked correctly for positive values, not negative variability. This is now fixed. 

New 3-point bilinear interpolation routine was not correctly computing the third point to use. This is now fixed. 

makefile no longer enforces an old version of netcdf libraries on cheyenne. 

Added a very simple python based post-processing tool to combine the mean, error, PoP, and SCRFs.

### CODE REVIEW

File: helpers/post_proc_gard.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+        # figure out where the probability was not exceeded
+        noprecip = rand_tmp.values < (1-popval.values)
+        # rescale the random numbers to account for the limited portion of the cdf they should occupy
+        rand_tmp.values -= (1-popval.values)
+        rand_tmp /= popval.values
+
+    # convert uniform random numbers to normally distributed values
+    rand_tmp.values[rand_tmp.values<=1e-20] = 1e-20
+    errors = stats.norm.ppf(rand_tmp.values)
```
Comment (jhamman) [inline helpers/post_proc_gard.py]: Ideally, we don't have to pull out the values here. I think you can stick with xarray/dask via something like this:

```Python
LOWER_THRESH = 1e-20
rand_tmp = rand_tmp.where(rand_tmp < LOWER_THRESH, LOWER_THRESH)
errors = ppf(rand_tmp)
```

You probably want to use the xarray ufunc version of ppf and cbrt available here: 

https://github.com/jhamman/storylines/blob/8bf2869ec665f9472adb6c9213b9b570bf85e8bf/storylines/tools/post_process_gard_output.py#L36-L45

File: helpers/post_proc_gard.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    if has_pop:
+        # load the PoP dataset
+        pop = xr.open_dataset(pop_file)
+        popval = pop[varname+"_exceedence_probability"]
+
+        # figure out where the probability was not exceeded
+        noprecip = rand_tmp.values < (1-popval.values)
+        # rescale the random numbers to account for the limited portion of the cdf they should occupy
+        rand_tmp.values -= (1-popval.values)
+        rand_tmp /= popval.values
```
Comment (jhamman) [inline helpers/post_proc_gard.py]: I'm wondering if we can just use what is in https://github.com/jhamman/storylines/blob/refactor/storylines/tools/post_process_gard_output.py#L79

That code does what this does, plus has some performance optimizations. I'd be totally fine moving that module over here.

File: src/makefile [hunk 3]
```diff
@@ -84,13 +87,13 @@ ifeq ($(NODENAME), cheyenne)
...
 		COMPILER=gnu
-        NCDF_PATH=/glade/u/apps/ch/opt/netcdf/4.4.1.1/gnu/6.2.0
+        # NCDF_PATH=/glade/u/apps/ch/opt/netcdf/4.4.1.1/gnu/6.2.0
         LAPACK_PATH=/glade/u/apps/ch/opt/netlib/3.7.0/gnu/4.8.5
     endif
 	ifeq ($(LMOD_FAMILY_COMPILER),intel)
 	    FC=ifort
 		COMPILER=intel
-	    NCDF_PATH=/glade/u/apps/ch/opt/netcdf/4.4.1.1/intel/17.0.1
+	    # NCDF_PATH=/glade/u/apps/ch/opt/netcdf/4.4.1.1/intel/17.0.1
```
Comment (jhamman) [inline src/makefile]: let's just remove these if they are no longer relevant.

File: helpers/post_proc_gard.py (hunk 1 above)
Comment (gutmann) [reply helpers/post_proc_gard.py]: Yes, if you have the time to simplify your script to remove the aspects related to, e.g., RCP dates and other storylines work, that would be great.  Unless you can do that today / tomorrow, then I’d say let’s merge this in, but plan on replacing it as soon as feasible with yours.  Maybe setting yours up to read the gard configuration file so it automatically uses the correct files / options.  Keep in mind that GARD output files are not CF compliant / xarray friendly... scrf files may not be.

Comment (gutmann) [reply helpers/post_proc_gard.py]: (That should have been a reply on the larger review comment, not this particular comment)

File: src/makefile (hunk 3 above)
Comment (gutmann) [reply src/makefile]: Agreed, that was just laziness on my part until I had a chance to test it.

**Pull request merged**